SummerProject/static/src/components/Sidebar/index.js

```diff
@@ -0,0 +1,87 @@
+import React from "react";
+import PropTypes from "prop-types";
+import axios from "axios";
+import "./style.css";
+
+class Sidebar extends React.Component {
+  state = {
+    category: [],
+    error: false
+  };
+
+  componentDidMount() {
+    axios.get("http://localhost:8000/api/v1/category/")
+      .then(res => {
+        const category = res.data;
```

Comment:
Please change `category`  to `categories` (or `categoriesList`)

Reply:
done